cert/spec/runner_spec.rb

```diff
@@ -73,5 +73,48 @@
       Cert.config = FastlaneCore::Configuration.create(Cert::Options.available_options, keychain_path: ".")
       Cert::Runner.new.revoke_expired_certs!
     end
+
+    describe ":filename option handling" do
+      input_filename = ""
+      let(:generate) do |options|
+        filename = input_filename
+        filename_ext = File.extname(filename) == ".cer" ? filename : "#{filename}.cer"
+        certificate = stub_certificate
+
+        allow(Spaceship).to receive(:login).and_return(nil)
+        allow(Spaceship).to receive(:client).and_return("client")
+        allow(Spaceship).to receive(:select_team).and_return(nil)
+        allow(Spaceship.client).to receive(:in_house?).and_return(false)
+        allow(Spaceship.certificate.production).to receive(:all).and_return([certificate])
+        allow(FastlaneCore::CertChecker).to receive(:installed?).and_return(true)
+
+        expect(File.exist?(filename_ext)).to be false
```

Comment:
Are you able to remove the `expect`s from this `let(:generate)` and move them into the `its` below? the `let` should really only be for creating, not testing 😊 

Other than that this looks really good 😊

Reply:
OoOoh I thought that you will tell me that 😁I did it for being identical to [fastlane/certs/commands_generator_spec#L4-L8](https://github.com/fastlane/fastlane/blob/master/cert/spec/commands_generator_spec.rb#L4-L8).

I have updated code, have a look 😉 FYI, I had to move an extra method for avoiding to many code duplicates.

Commits rebased : 🆗